server/src/main/java/org/elasticsearch/action/support/broadcast/node/TransportBroadcastByNodeAction.java

```diff
@@ -610,8 +619,9 @@ public void writeTo(StreamOutput out) throws IOException {
     }
 
     /**
-     * Can be used for implementations of {@link #shardOperation(BroadcastRequest, ShardRouting, Task, ActionListener) shardOperation} for
-     * which there is no shard-level return value.
+     * Can be used for implementations of
+     * {@link #shardOperation(BroadcastRequest, ShardRouting, Task, Object, ActionListener) shardOperation} for which there is no
+     * shard-level return value.
```

Comment:
nit: no need for args here

```suggestion
     * Can be used for implementations of {@link #shardOperation} for which there is no shard-level return value.
```